src/components/itemModal.html

```diff
@@ -53,6 +53,10 @@
                                         <img src="assets/images/items/Trade_stone.png">
                                         <div data-bind="text: player._itemList['Trade_stone']()"></div>
                                     </div>
+									<div class=col style="cursor: pointer" data-bind="css:{ 'item-selected': ItemHandler.stoneSelected() === 'Sun_stone'}, click: function(){ ItemHandler.stoneSelected('Sun_stone'); ItemHandler.pokemonSelected('')}">
+									<img src=assets/images/items/Sun_stone_small.png>
+									<div data-bind="text: player._itemList['Sun_stone']()"/>
```

Comment:
Could you please close this div, unfortunately [div elements do not allow self closing](https://stackoverflow.com/questions/7971716/is-it-ok-to-use-a-self-closing-div-tag).
```suggestion
                                        <div data-bind="text: player._itemList['Sun_stone']()"></div>
```